src/main.rs

```diff
@@ -52,6 +56,12 @@ enum Connection {
     Open(Server),
 }
 
+impl Connection {
+    fn is_open(&self) -> bool {
+        matches!(self, &Self::Open(_))
+    }
+}
+
```

Comment:
Nie używasz tego nigdzie

Reply:
[używam](https://github.com/mimuw-jnp2-rust/small-task-2-communications-ggegoge/blob/ee21fe01c832bf794daaf956514e2f06cdc3e576/src/main.rs#L139)

Reply:
Ah, faktycznie, to nie jest `is_open` z klienta. Zwracam honor